src/sort_cards/sort_cards.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+
+def sort_cards(cards):
+    """Sort shuffled list of cards, sorted by rank.
+
+    >>> sort_cards(['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K'])
+    ['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']
+
+    """
+
+    new_list = []
+    for i, val in enumerate(cards):
+        if val == 'A':
```

Comment:
Seems like you could've used a dictionary to replace these massive if-elif-else blocks